yaksh/stdio_evaluator.py

```diff
@@ -0,0 +1,22 @@
+class Evaluator(object):
+
+    def evaluate(self, user_answer, proc, expected_input, expected_output):
```

Comment:
add a default setup() and teardown() here which is overridden by the child classes